libs/sdk-core/src/lib.rs

```diff
@@ -1,3 +1,143 @@
+//! # Breez SDK
+//!
+//! The Breez SDK makes it easy to build non-custodial Bitcoin and Lightning applications.
+//!
+//! The Greenlight integration allows it to separate the running of a LN node from the signing of LN
+//! transactions. The node runs in the cloud and is ready to receive payments non-custodially,
+//! while the caller of the SDK has full custodial access to these funds and can sign spending transactions
+//! locally.
+//!
+//! On initializing the SDK, the caller gets its [GreenlightCredentials]. These are used to interact
+//! with the Greenlight LN node running in the cloud. Together with the BIP39 mnemonic, these can be used to
+//! restore access to the same cloud node, either in the same app (backup / restore) or in another app
+//! using the SDK.
+//!
+//! In addition, the SDK includes utilities for LSP management, on-chain payments and fiat on- and
+//! off-ramps.
+//!
+//! ## Getting Started
+//!
+//! The first step is to initialize the SDK and start the node:
+//!
+//! ```ignore
+//! let mnemonic = Mnemonic::new(Words12, English);
+//! let seed = Seed::new(&mnemonic, "");
+//!
+//! let creds = BreezServices::register_node(Network::Bitcoin, seed.as_bytes().to_vec()).await?;
+//! let sdk = BreezServices::init_services(
+//!         None,
+//!         seed.to_vec(),
+//!         creds.clone(),
+//!         Box::new(AppEventListener {}),
```

Comment:
I think in the following sections where you put some great details about the possibilities it worth adding a section for the event listener and which events can be consumed by the user.

Reply:
Done, I added a paragraph with links in the intro section.